### PR TITLE
documentation: align all examples to a single location

### PR DESCRIPTION
**Pull request by katbyte**

### CODE REVIEW

File: website/docs/r/virtual_network_gateway_connection.html.markdown [hunk 1]
```diff
@@ -92,7 +92,7 @@ in different locations/regions.
 ```hcl
 resource "azurerm_resource_group" "us" {
   name     = "us"
-  location = "East US"
+  location = "West Europe"
```
Comment (favoretti) [inline website/docs/r/virtual_network_gateway_connection.html.markdown:95]: These kind of examples will be tricky to understand...

Comment (favoretti) [reply website/docs/r/virtual_network_gateway_connection.html.markdown:95]: Namely, this demonstrates peering/gateways between regions and now they are both referring to the same region.

**Review by tombuildsstuff — APPROVED** [review]
LGTM 👍

File: website/docs/r/virtual_network_gateway_connection.html.markdown (hunk 1 above)
Comment (tombuildsstuff) [reply website/docs/r/virtual_network_gateway_connection.html.markdown]: looks like this has been resolved

**Pull request merged**